### DISCUSSION TITLE
Problems with the reader mode

### DISCUSSION THREAD

**Discussion by Ypot** [plateaukao/einkbro#436] [General]
There are some webs that can't be read in the reader mode, for example this one: 
https://git.savannah.gnu.org/cgit/emacs.git/tree/etc/NEWS?h=emacs-30

**Comment by plateaukao** [comment]
@Ypot 
For the examples you provided, you could just click link "plain", it's just pure text already.
<img width="462" alt="image" src="https://github.com/user-attachments/assets/886fc48a-3a67-41ec-91b7-e01e79c3eb11">

**Reply by Ypot** [reply]
Yet text doesn't adjust, does it?
